packages/env/lib/config.js

```diff
@@ -255,6 +255,31 @@ function includeTestsPath( source ) {
 	};
 }
 
+/**
+ * Gets a port number environment variable.
+ *
+ * Throws an error if the variable cannot be parsed to a number.
+ * Returns null if the environment variable has not been specified.
+ *
+ * @param {string} varName The environment variable to check (i.e. WP_ENV_PORT)
+ * @return {null|number} The port number. Null if it does not exist.
+ */
+function getEnvPortOverride( varName ) {
```

Comment:
Why is this named specifically for ports but takes an arbitrary variable name? It makes me think we need a generic function for any environment variable or one function for each port.

Reply:
Since the behavior would be the same for any number-based value, we could change it to `getNumberFromEnvVariable`